partner_relations/models/res_partner_relation_type_selection.py

```diff
@@ -1,149 +1,128 @@
 # -*- coding: utf-8 -*-
 # © 2014-2016 Therp BV <http://therp.nl>
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
-'''
-Created on 23 may 2014
-
-@author: Ronald Portier, Therp
-
-rportier@therp.nl
-http://www.therp.nl
-
+"""
 For the model defined here _auto is set to False to prevent creating a
-database file. All i/o operations are overridden to use a sql SELECT that
-takes data from res_partner_connection_type where each type is included in the
+database file. The model is based on a SQL view based on
+res_partner_relation_type where each type is included in the
 result set twice, so it appears that the connection type and the inverse
 type are separate records..
 
 The original function _auto_init is still called because this function
 normally (if _auto == True) not only creates the db tables, but it also takes
 care of registering all fields in ir_model_fields. This is needed to make
 the field labels translatable.
-
-example content for last lines of _statement:
-select id, record_type,
-  customer_id, customer_name, customer_city, customer_zip, customer_street,
-  caller_id, caller_name, caller_phone, caller_fax, caller_email
-from FULL_LIST as ResPartnerRelationTypeSelection where record_type = 'c'
-ORDER BY ResPartnerRelationTypeSelection.customer_name asc,
-ResPartnerRelationTypeSelection.caller_name asc;
-
-'''
+"""
 from psycopg2.extensions import AsIs
+
 from openerp import api, fields, models
 from openerp.tools import drop_view_if_exists
+
 from .res_partner_relation_type import ResPartnerRelationType
-from .res_partner import PADDING
+
+
+PADDING = 10
```

Comment:
I think this is going to bite us if we ever want/need to change this. Why don't you import it from partner_relation_all? Given you were busy with this anyways, how about renaming the variable (technically, this is not a padding but a multiplicator)? When doing my corrections, I was tempted to rename it then already, but decided against it for the sake of small diffs.
We can also consider changing the value to 2 in order to save id space - now, this module can only handle ids in the range of MAX_INT/10, while MAX_INT/2 would be possible. Given it's a view, no migration should be needed.